esi_leap/tests/resource_objects/test_ironic_node.py

```diff
@@ -197,3 +210,11 @@ def test_get_node_cache(self, mock_gn):
         self.assertEqual(fake_get_node,
                          test_ironic_node._get_node())
         mock_gn.assert_not_called
+
+    @mock.patch('esi_leap.common.ironic.get_node')
+    def test_get_unknown_node(self, mock_gn):
+        unknown_get_node = UnknownIronicNode()
+        mock_gn.return_value = None
```

Comment:
Ah, this isn't exactly what we want. We want something like:

```
mock_gn.side_effect = exceptions.NotFound
```

This will result in this test testing _get_node along the exception path. The syntax above may not be quite exact; you can search for 'side_effect' in the test cases to see a more exact guide!